dowhy/causal_estimators/two_stage_regression_estimator.py

```diff
@@ -6,7 +6,7 @@
 
 from dowhy.causal_estimator import CausalEstimate, CausalEstimator
 from dowhy.causal_estimators.linear_regression_estimator import LinearRegressionEstimator
-from dowhy.causal_identifier import CausalIdentifier
+from dowhy.causal_identifier.identify_effect import CausalIdentifierEstimandType
```

Comment:
we can simply call it CausalEstimandType. We do not need Identifier. 

In fact, since dowhy is about causality, EstimandType would also work.

Reply:
Would you say this should be defined in a more generic package instead of causal_identifier so that it can be reused in a more general way across all of dowhy?

Reply:
Let's keep it where it is right now.  The current EstimandType is specific to the effect inference pipeline and the interface between identification and estimation.